services/director-v2/src/simcore_service_director_v2/utils/exceptions.py

```diff
@@ -0,0 +1,17 @@
+from typing import Optional
+
+from models_library.projects import ProjectID
+
```

Comment:
Consider moving these errors from ``utils``:  ``ProjectNotFoundError`` should go to ``db.errors`` and director exception to ``modules/director_v0``?